src/Core/Layer/Layer.js

```diff
@@ -1,5 +1,77 @@
 import { EventDispatcher } from 'three';
 
+/**
+ * Layer's sequence (index) event
+ * @event Layer#sequence-property-changed
+ * @property layerId {string} the id of the layer
+ * @property new {object}
+ * @property new.sequence {number} the new value of the layer sequence
+ * @property previous {object}
+ * @property previous.sequence {number} the previous value of the layer sequence
+ * @property target {Layer} dispatched on layer
+ * @property type {string} sequence-property-changed
+ * @example
```

Comment:
I think this example is too trivial to be interesting. Do we need to explain to js dev how to call `addEventListener`? What devs will look for is the name of the possible events, and what properties these `event` contains when they fire.